README.md

```diff
@@ -16,13 +16,13 @@ Then start Rocket.Chat linked to this mongo instance:
 
     docker run --name rocketchat --link db -d rocket.chat
 
-This will start a Rocket.Chat instance listening on the default Meteor port of 3000 on the container.
+This will start a Rocket.Chat instance listening on the default Meteor port of 3000 on the container.  You can then link a web server to front-end Rocket.Chat.
```

Comment:
Having said that, I'm okay with going ahead per your suggestion - and then mod as necessary based on feedback.